cpp/include/Ice/Instrumentation.h

```diff
@@ -1058,40 +378,16 @@ class ICE_API CommunicatorObserver : public virtual ::Ice::LocalObject
      * @param size The size of the dispatch.
      * @return The dispatch observer to instrument the dispatch.
      */
-    virtual DispatchObserverPtr getDispatchObserver(const ::Ice::Current& c, ::Ice::Int size) = 0;
+    virtual DispatchObserverPtr getDispatchObserver(const ::Ice::Current& c, int size) = 0;
 
     /**
      * The Ice run-time calls this method when the communicator is initialized. The add-in implementing this
      * interface can use this object to get the Ice run-time to re-obtain observers for observed objects.
      * @param updater The observer updater object.
      */
-    virtual void setObserverUpdater(const ObserverUpdaterPtr& updater) = 0;
+    virtual void setObserverUpdater(const  ObserverUpdaterPtr& updater) = 0;
```

Comment:
An extra space was added here
```suggestion
    virtual void setObserverUpdater(const ObserverUpdaterPtr& updater) = 0;
```